src/endpoints/metrics.js

```diff
@@ -26,6 +26,19 @@ const getResourceQuery = (resourceType) => {
   return { types: { $regex: ".*" } }
 }
 
+/**
+ * Returns the mongo query to use for the specified date range
+ * @param {string} startDate The start date
+ * @param {string} endDate The end date
+ * @returns {object} The query to send to mongo to filter by the specified dates
+ */
+const getDateQuery = (startDate, endDate) => {
+  return {
+    $gt: new Date(startDate),
+    $lt: new Date(endDate),
+  }
+}
+
```

Comment:
Small refactor to DRY this up, since we now use this filter in three different endpoints